engfrosh_site/management/views.py

```diff
@@ -424,200 +419,6 @@ def manage_frosh_teams(request: HttpRequest) -> HttpResponse:
     return HttpResponseBadRequest()
 
 
-@permission_required("discord_bot_manager_discordchannel.change_discordchannel")
-def manage_discord_channels(request: HttpRequest) -> HttpResponse:
-    """Page for managing discord channels, such as locking and unlocking."""
-
-    if request.method == "GET":
-
-        context = {}
-
-        channels = DiscordChannel.objects.all()
-        context["channels"] = channels
-
-        return render(request, "discord_channels.html", context)
-
-    else:
-
-        return HttpResponseBadRequest("Bad http request method.")
-
-
-@permission_required("common_models.view_discordguild")
-def manage_discord_servers(request: HttpRequest) -> HttpResponse:
```

Comment:
The guild creation can be removed, the one use invite I found very handy when getting dozens of emails asking for links to the discord, so you may want to keep that functionality (since you @finlaywashere will be the one dealing with those emails :) ) but I don't care where it goes.

Reply:
Isn't emailing users the link to the discord redundant as the site already adds users to the guild when they link their discord account?

Reply:
That flow fails sometimes, for some reason. I don't know why, but every year there are a number of people who can't use the magic link. It wasn't too bad this past year (real bad in 2021), but it's something to consider how it will work on the edgecases, because with over 1000 people using it we will find some edgecases.